### PR TITLE
Find center and find angle

### PR DESCRIPTION
**Pull request by gleich**
findCenter and findAngle functions filled in the file image_calculations.py

### CODE REVIEW

File: image_calculations.py [hunk 1]
```diff
@@ -2,28 +2,34 @@
...
+        cx1 -> float center coordinates of first rectangle
+        cx2 -> float center coordinates of second rectangle
     Output:
         angle -> float
 
     This function finds the robot's angle relative to the center of the target
-
     """
-    pass
+    pixel_along_diagoal = sqrt(camera_res[0] ** 2 + camera_res[1] ** 2)
```
Comment (ionigman-501) [inline image_calculations.py]: The math used here isn't the proper math for a camera FOV. There was a link i posted as a hint in the channel. It's some pretty complex stuff beyond the pythagorean theorem. Take a look at the hints links, and re-work this to get the horizontal FOV coefficient, which you can then multiply the pixel offset by.

File: image_calculations.py [hunk 2]
```diff
@@ -2,28 +2,34 @@
...
+    return angle_from_cnt
 
 def findCenter(cnt):
     """
     Find the center coordinates of a given contour cnt
     :param cnt: the contour of the target
     :return: the center x coordinate (cx), the center y coordinate (cy)
     """
-    pass
+    m = cv2.moments(cnt)
```
Comment (ionigman-501) [inline image_calculations.py]: math needs to be done on m to find the center of mass. m0, m1 etc. look at the hints i linked.

File: image_calculations.py [hunk 3]
```diff
@@ -2,28 +2,34 @@
 import math
 from collections import Counter
 
+camera_res = [320, 240]
+camera_degree = 60
```
Comment (ionigman-501) [inline image_calculations.py]: Camera diagonal FOV is 60

**Review by ionigman-501 — CHANGES_REQUESTED** [review]
take a look at comments

**Comment by gleich** [comment]
Ok, I will work on these things throughout the day.

File: image_calculations.py (hunk 1 above)
Comment (Matt-Gleich) [reply image_calculations.py]: What is the pixel offset?

**Comment by gleich** [comment]
Just filled in both functions and should be all good now. 👍

**Pull request merged**